.github/workflows/maven.yml

```diff
@@ -0,0 +1,23 @@
+name: Java CI with Maven
```

Comment:
[INFO] Skipping JaCoCo execution due to missing execution data file:/home/runner/work/VershininMaxim/VershininMaxim/target/jacoco.exec